src/_scss/pages/redirect/redirectModal.scss

```diff
@@ -0,0 +1,51 @@
+@import "all";
```

Comment:
Why is this not in the `modals` directory we have in `_scss/pages`?

Reply:
Moved to the modals directory